config/eirini/enable-automount-service-account-token.yml

```diff
@@ -1,7 +1,8 @@
+#@ load("@ytt:data", "data")
```

Comment:
Happy to make these align for now. As I stated, I believe @Syerram is planning to revisit the data values interface soon, so this could all change again next week.

Reply:
The only one that this doesn't work with is `enable_load_balancer`, which controls the contents of `config/networking/ingressgateway-service-nodeport.yml`.